### ISSUE TITLE
Automatically Remove Notifications When Opening a Task

### ISSUE THREAD

**Issue by akmalkadi** [kanboard/kanboard#5565]
Currently, notifications are only deleted when clicked. However, they should also be deleted automatically when the task is opened and read.

There should be an option or plugin that allows all notifications related to a task to be deleted once the task is opened.

**Comment by tomeli5n** [comment]
maybe https://github.com/tomeli5n/NotifyPlus/ will work for your use case.
